addons/material-forum-editor-buttons/addon.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "Material forum editor buttons",
+  "description": "Changes the forum editor icons from tiny 16x16 bitmaps to SVG Google Material Icons.",
+  "userstyles": [
+    {
+      "url": "icons.css",
+      "matches": ["https://scratch.mit.edu/discuss/*"]
+    }
+  ],
+  "tags": ["community", "forums", "theme", "beta", "recommended"],
+  "credits": [
+  {
+    "name": "easrng",
+    "link": "https://github.com/easrng/material-scratch-forum-editor-buttons"
+  },
+  {
+    "name": "Secret-chest",
+    "link": "https://github.com/Secret-chest"
+  }
+  ],
+  "enabled_by_default": false
```

Comment:
This should be `"enabledByDefault": false`. That is why schema validation is failing.